src/site/markdown/usage.md

```diff
@@ -24,11 +24,23 @@ Once extension is activated, cache will kick-in automatically on every lifecycle
 Build could be invoked on any module in project and will try to discover cache by introspecting dependencies. In order
 to identify which dependencies are part of cacheable project the cache engine needs to know:
 
-* full project root location which must be passed with `-Dmaven.multiModuleProjectDirectory`
-* Specify profiles which activate full graph in config:
+* Full project root location which must be passed with `-Dmaven.multiModuleProjectDirectory`
+* Profiles which activate full graph in project build. Underlying implementation logic is to introspect reactor
+  graph in `full` mode to discover which dependencies are part of the cacheable project. This information will be used
+  then any subtree is being build to identify dependencies as a part of the project:
```

Comment:
Not sure of the intended meaning of `will be used then`...

Reply:
Thank you! Slightly reworded, hope it is better now!